src/frontend/src/showcase.ts

```diff
@@ -248,13 +248,18 @@ const iiPages: Record<string, () => void> = {
         },
       },
     }),
-  verifyDevice: () =>
-    verifyDevice(
-      userNumber,
-      dummyConnection,
-      simpleDevices[0],
-      undefined as unknown as bigint
-    ),
+  verifyTentativeDevice: () =>
+    verifyTentativeDevicePage({
+      alias: simpleDevices[0].alias,
+      cancel: () => console.log("canceled"),
+      verify: () => Promise.resolve({ retry: null }),
+      doContinue: (v) => console.log("continue with:", v),
+      remaining: {
+        async *[Symbol.asyncIterator]() {
+          yield "00:34";
```

Comment:
aaaaaah! I had a look at the screenshots and I was like "interesting, now it takes 34 seconds to take the screenshots" 😁